receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -388,6 +407,12 @@ func (jr *jReceiver) startCollector(ctx context.Context, host component.Host) er
 			return err
 		}
 
+		jr.settings.Logger.Info("Starting HTTP collector server",
```

Comment:
HTTP server for what? "collector" is meaningless here.

Reply:
I should've rightly specified there.

Updated in latest commit.